athlinks-schema.ttl

```diff
@@ -6,13 +6,24 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix schema: <http://schema.org/> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://adp.athlinks.com/2017/01/rdf-schema> a owl:Ontology ;
   owl:imports <http://topbraid.org/schema/> ;
   rdfs:label "The Athlinks Schema vocabulary".
 
 ### CLASSES
+athlinks:TerrainSpecifics a rdfs:Class;
+  rdfs:comment "Specifics about course terrain such as altitude or grade." ;
+  rdfs:label "Terrain Specifics" ;
+  rdfs:subClassOf schema:Landform ;
+  #members:
+  #athlinks:elevUnit;
+  #athlinks:altitude;
+
+athlinks:Directions a rdfs:Class;
+  rdfs:comment "Directions to an event or race." ;
+  rdfs:label "Directions" ;
+  rdfs:subClassOf schema:Text ;
```

Comment:
I think this class would be better as a property with a domain of schema:Event and a range of schema:Text.